code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -99,9 +99,12 @@
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/device/soulstone))
 		var/obj/item/device/soulstone/SS = O
+		if(!iscultist(user) && !iswizard(user))
```

Comment:
Nope. I made some construsts in non-cult rounds and they weren't slaved to nar-sie. They were murdering some of the crew, but i'm sure it was self-defense.